storagetransfer/api/StorageTransfer.Samples.Tests/QuickstartTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Google.Cloud.StorageTransfer.V1;
+using Xunit;
+
+namespace StorageTransfer.Samples.Tests
+{
+    [Collection(nameof(StorageFixture))]
+    public class QuickstartTest
+    {
+        private readonly StorageFixture _fixture;
+
+        public QuickstartTest(StorageFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [Fact]
+        public void TestQuickstart()
+        {
+            QuickstartSample quickstartSample = new QuickstartSample();
+            var transferJob = quickstartSample.Quickstart(_fixture.ProjectId, _fixture.BucketNameSource, _fixture.BucketNameSink);
+            Assert.Contains("transferJobs/", transferJob.Name);
+
+            // Delete the job to clean up
```

Comment:
Have this class implement IDisposable and move this code there in a try...catch. That way the test won't fail if the deletion fails.